doc/source/basics.rst

```diff
@@ -1788,6 +1788,46 @@ but occasionally has non-dates intermixed and you want to represent as missing.
 In addition, :meth:`~DataFrame.convert_objects` will attempt the *soft* conversion of any *object* dtypes, meaning that if all
 the objects in a Series are of the same type, the Series will have that dtype.
 
+:meth:`~pandas.to_numeric` is very similar to `pd.to_datetime`, except that it is for conversion to numerical data:
+
+.. ipython:: python
+
```

Comment:
I think I'll leave it as is for the time being.  The whole section is about object conversion.